backend/src/app.ts

```diff
@@ -29,7 +29,8 @@ import { connectToPostgreSql } from './shared/infrastructure/repository/connectT
 import { PostgreSqlTournamentRegistrationsRepository } from './modules/tournaments-registrations/infrastructure/repository/postgresql/PostgreSqlTournamentRegistrationsRepository';
 import { PlayerProfilesModuleCore } from './modules/player-profiles/core/PlayerProfilesModuleCore';
 import { PlayerProfileRestApiModule } from './modules/player-profiles/presentation/rest-api/PlayerProfileRestApiModule';
-import { InMemoryPlayerProfileRepository } from './modules/tournaments-registrations/infrastructure/repository/inmemory/InMemoryPlayerProfileRepository';
+import { InMemoryPlayerProfileRepository } from './modules/player-profiles/infrastructure/repository/inmemory/InMemoryPlayerProfileRepository';
```

Comment:
Teraz skoro moduł jest skończony, to możesz usunąć:
initializeDummyData(). 
Albo zmienić to ewentualnie na wywołanie odpowiednich command :) 

Mój moduł po prostu zależał od eventu, więc go tutaj generowałem, pomijając cały flow modułu z profilami:
![image](https://user-images.githubusercontent.com/18637116/109710152-a009a280-7b9d-11eb-8ed0-c86e90fe1692.png)



Teraz wyjdzie na to samo, jak się da command i nie będzie niespójności między modułami. Bo samo opublikowanie eventu skutkuje handlowaniem i zapisaniem playera tylko na potrzeby tournament registrations i mamy rozjazd w tym momencie.

Jesli za to zrobisz execute Command, to wtedy command i zapisze u Ciebie, poskutkuje eventem, który też zapisze co trzeba w module tournament registartions. Jasne :) Na Event Modelingu to łatwo widać.

![Uploading image.png…]()

Reply:
Ale rownie dobrze mozesz na to osobny PR zrobic, zeby tego nie kisic :)

Reply:
Pokombinuje z tym w kolejnym PR, bo kończę metodę GET i wtedy będzie domknięcie całego modułu :-)

Reply:
Rozumiem, że wystarczyło to usunąć, Si? 
@nowakprojects